package.json

```diff
@@ -45,6 +46,7 @@
   "scripts": {
     "postinstall": "npm run build-css && node postinstall.js",
     "start": "concurrently \"npm run watch-css\" \"electron .\"",
+    "dev": "concurrently \"npm run watch-css\" \"NODE_ENV=development electron .\"",
```

Comment:
You could add a note about this in the readme, and how it differs from `npm start`. You said `npm run-script dev` in the PR, but I think `npm run dev` does the same.